backend/middlewares/auth.ts

```diff
@@ -29,6 +29,24 @@ export const getAccessToken = (req: Request): string | null => {
   return null;
 };
 
+/* No-op authorization check which always resolves */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export const isAuthorizedForEveryone =
```

Comment:
what's this for

Reply:
Eventually, we should probably add access control to all our queries so that unauthenticated users can't access random data. To make sure we don't miss any, I want to add Typescript-enforced checks so that every single query requires us setting access control. This would be used for queries that are truly public

Reply:
No particular reason I put it in this PR, mostly just so I don't forget how to do this